docs/tutorials/05-Experiment_Tracking.ipynb

```diff
@@ -154,7 +154,6 @@
     "    auto_lr_find=True, # Runs the LRFinder to automatically derive a learning rate\n",
     "    batch_size=1024,\n",
     "    max_epochs=100,\n",
-    "    gpus=-1,  #index of the GPU to use. -1 means all available GPUs, None, means CPU\n",
```

Comment:
dropping all these in favor of `accelerator="auto"` which is already in config default :rabbit: